specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/examples/BlobContainersPutObjectLevelWorm.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
```

Comment:
Is there a region enabled for VLW.
I tried to create an account with VLW enabled in account in canary (eastus2euap), but failed with 400.

```
PUT https://management.azure.com/subscriptions/{subscription-id}/resourceGroups/***/providers/Microsoft.Storage/storageAccounts/***/blobServices/default/containers/container7176?api-version=2021-04-01 HTTP/1.1
x-ms-client-request-id: 1cbed7ec-0d47-478d-a684-68faec042d88
Accept-Language: en-US
Authorization: Bearer ***
User-Agent: FxVersion/4.6.29916.01 OSName/Windows OSVersion/Microsoft.Windows.10.0.19042. Microsoft.Azure.Management.Storage.StorageManagementClient/21.1.0.0
Content-Type: application/json; charset=utf-8
Content-Length: 99
Host: management.azure.com

{
  "properties": {
    "immutableStorageWithVersioning": {
      "enabled": true
    }
  }
}
HTTP/1.1 400 Bad Request
Cache-Control: no-cache
Pragma: no-cache
Content-Length: 131
Content-Type: application/json
Expires: -1
x-ms-request-id: aa1532ce-1409-4ad4-b1d6-4b91f4431f5f
Strict-Transport-Security: max-age=31536000; includeSubDomains
Server: Microsoft-Azure-Storage-Resource-Provider/1.0,Microsoft-HTTPAPI/2.0 Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-writes: 1194
x-ms-correlation-request-id: 67dc00fc-b0b2-4173-b54e-bbf9bcb1720e
x-ms-routing-request-id: SOUTHEASTASIA:20210428T092733Z:67dc00fc-b0b2-4173-b54e-bbf9bcb1720e
X-Content-Type-Options: nosniff
Date: Wed, 28 Apr 2021 09:27:32 GMT

{"error":{"code":"FeatureNotAvailableInRegion","message":"ImmutableStorageWithVersioning feature is not available in this region"}}
```